source/type/pb_converter.h

```diff
@@ -51,15 +51,21 @@ namespace ng = navitia::georef;
 enum class DumpMessage : bool { Yes, No };
 
 enum class DumpLineSectionMessage : bool { Yes, No };
+enum class DumpRailSectionMessage : bool { Yes, No };
 
 struct DumpMessageOptions {
     DumpMessage dump_message;
     DumpLineSectionMessage dump_line_section;
+    DumpRailSectionMessage dump_rail_section;
     const nt::Line* line;  // specific line to retrieve only related line-sections
     constexpr DumpMessageOptions(DumpMessage dump_message = DumpMessage::Yes,
                                  DumpLineSectionMessage dump_line_section = DumpLineSectionMessage::No,
-                                 const nt::Line* line = nullptr)
-        : dump_message(dump_message), dump_line_section(dump_line_section), line(line) {}
+                                 const nt::Line* line = nullptr,
+                                 DumpRailSectionMessage dump_rail_section = DumpRailSectionMessage::No)
+        : dump_message(dump_message),
+          dump_line_section(dump_line_section),
+          dump_rail_section(dump_rail_section),
```

Comment:
We should avoid using same name for argument and member variable, a good practice is to add _ at the end of member variable